app/models/validations/shared_validations.rb

```diff
@@ -64,20 +64,22 @@ def validate_property_postcode(record)
     end
   end
 
-  def location_during_startdate_validation(record, field)
+  def location_during_startdate_validation(record)
     location_inactive_status = inactive_status(record.startdate, record.location)
 
     if location_inactive_status.present?
       date, scope, deactivation_date = location_inactive_status.values_at(:date, :scope, :deactivation_date)
-      record.errors.add field, :not_active, message: I18n.t("validations.setup.startdate.location.#{scope}", postcode: record.location.postcode, date:, deactivation_date:)
+      record.errors.add :startdate, :not_active, message: I18n.t("validations.setup.startdate.location.#{scope}.startdate", postcode: record.location.postcode, date:, deactivation_date:)
+      record.errors.add :location_id, :not_active, message: I18n.t("validations.setup.startdate.location.#{scope}.location_id", postcode: record.location.postcode, date:, deactivation_date:)
```

Comment:
we never add this validation to :scheme_id, although we used to? I suspect we did it because location id is sometimes inferred (?)